autowarefoundation/autoware
# [Feature] incremental voxel update for ndt_mapping

## Status
**DEVELOPMENT**

## Description
incremental voxel update for ndt_mapping  autowarefoundation/autoware_ai#53 
This feature was made by @anhnv3991.


## Related PRs

## Todos
- [] Documentation


## Steps to Test or Reproduce

1. Select "pcl_anh" and check  "incremental voxel update" from the app button of ndt_mapping.

1. launch ndt_mapping.

Be careful as it will consume a lot of memory when build a large map.
